lib/utils/fs.go

```diff
@@ -247,6 +247,10 @@ func overwriteFile(filePath string) error {
 		size += block
 	}
 
-	_, err = io.CopyN(f, rand.Reader, size)
-	return trace.Wrap(err)
+	if _, err := io.CopyN(f, rand.Reader, size); err != nil {
+		f.Close()
```

Comment:
Ditto for checking the Close error and doing it outside of this block.

Reply:
Moved this into the defer.